### PR TITLE
Make CVar RenderFOVColor settable by server only

### PR DESCRIPTION
**Pull request by notquitehadouken**
changing it lets you see in the dark

image

### CODE REVIEW

File: Robust.Shared/CVars.cs [hunk 1]
```diff
@@ -899,7 +899,7 @@ protected CVars()
             CVarDef.Create("render.sprite_direction_bias", -0.05, CVar.ARCHIVE | CVar.CLIENTONLY);
 
         public static readonly CVarDef<string> RenderFOVColor =
-            CVarDef.Create("render.fov_color", Color.Black.ToHex(), CVar.ARCHIVE | CVar.CLIENTONLY);
+            CVarDef.Create("render.fov_color", Color.Black.ToHex(), CVar.ARCHIVE | CVar.REPLICATED | CVar.SERVER);
```
Comment (PJB3005) [inline Robust.Shared/CVars.cs]: Make it not `ARCHIVE` if you are changing this.

**Comment by PJB3005** [comment]
This shouldn't have been a CVar at all IMO but I'm willing to accept a bandaid fix.

File: Robust.Shared/CVars.cs (hunk 1 above)
Comment (notquitehadouken) [reply Robust.Shared/CVars.cs]: ok

**Comment by notquitehadouken** [comment]
> This shouldn't have been a CVar at all IMO but I'm willing to accept a bandaid fix.

well in that case i can just abolish it

**Comment by PJB3005** [comment]
I didn't say the feature should've been entirely removed...

**Comment by notquitehadouken** [comment]
> I didn't say the feature should've been entirely removed...

bruh

File: Robust.Shared/CVars.cs (hunk 1 above)
Comment (PJB3005) [inline Robust.Shared/CVars.cs]: Now this is still ARCHIVE.

Comment (notquitehadouken) [reply Robust.Shared/CVars.cs]: GOD DAMMIT

**Comment by PJB3005** [comment]
PR closed for webedit.

**Comment by notquitehadouken** [comment]
> PR closed for webedit.

:trollface:

**Pull request merged**